server/server_test.go

```diff
@@ -7751,6 +7756,7 @@ stream
 		.message('message')
 		.details('details')
 		.crit(lambda: TRUE)
+		.sensu()
```

Comment:
What is this change doing? The alert should be sent to sensu via the topic handler, not explicitly in the TICKscript.